src/GafferSceneUI/LightPositionTool.cpp

```diff
@@ -669,16 +671,18 @@ void LightPositionTool::updateHandles( float rasterScale )
 	if( !m_drag )
 	{
 		bool singleSelection = selection().size() == 1;
+		bool isLight = s.scene()->set( g_lightsSetName )->readable().match( s.path() ) == IECore::PathMatcher::ExactMatch;
```

Comment:
It'd be unlikely, but it is possible to parent a light under another light. So I think we want `& ExactMatch` rather than `== ExactMatch` here.

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5723/commits/47d14183b7c818a0cc9b864110ace36d8bc88955